### DISCUSSION TITLE
RuntimeError: Weight proof did not arrive in time from peer

### DISCUSSION THREAD

**Discussion by Jily-Zhu** [Chia-Network/chia-blockchain#7059] [Support]
2021-06-30T14:40:52.549 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia\full_node\full_node.py", line 640, in _sync
RuntimeError: Weight proof did not arrive in time from peer: 76.201.87.186

**Comment by devtestnet** [comment]
- Fddgictw@@@is ax JN tesgg th j egg ujj in n hmm zuarad

**Reply by loppefaaret** [reply]
listen @devtestnet if you don't start pulling yourself together, and start posting _anything_ remotely understandable, we are going to have to remove your access to our github!

**Reply by devtestnet** [reply]
Please do so

**Comment by loppefaaret** [comment]
just means it had a timeout waiting for data, from that peer, it will try another peer - try and give it some time to get it all fixed up!

**Reply by Jily-Zhu** [reply]
So this problem is not serious. I can wait and see, right?

**Reply by Jily-Zhu** [reply]
But I have many full nodes with the same problem, all staying at the height of 480666

**Reply by loppefaaret** [reply]
yes, give it a bit of time, and see if it cant get pass this it self - the alternative is very disruptive

**Reply by loppefaaret** [reply]
do you have any that are passed 480666 ? you could close down one that are passed that block, and synced, and take a copy of its blockchain database, and distribute it to other clients manually

**Reply by Jily-Zhu** [reply]
No, all of them stay at the same height, 480666

**Reply by loppefaaret** [reply]
damn... ok, i hope you get one of them pass it 🤞

**Reply by Jily-Zhu** [reply]
thank you for your kindness

**Comment by devtestnet** [comment]
[Bahachina was the 1 to send me the bot trading concept with some fillers users but I didn't used it as my friend MR Dinesh Sharma is already into this type of seen.]()


Just want and information that what ia this happening.

Can do hard work but I feel that people are trading and swapping their nearer and dearer @. I cannot do this work. So I request you to be straight and clear. git GOT community support and then I am coming backwards.

And it's any special warning for all those who are doing all this.

Playing with an emotion of humanity is just and will be infront of mtggyf cry sjeuctcfhsapwmnmbzbbdhrukwb

Reply stop ✋ must 

    As if my mentors SAT @IN DOWN A CHAIR AND U  WILL KNOW the all platform will fucked off.


My name is Deepak Gusain. And will not hide from anyone.


And one more thing cross platform should be known understand before start. Ink think of sink.. Jai Mata Di!
.  BB dbs d bhej. Na s 

Please go further with my thought blessed by you all @ almighty.

- [ ] uuuqsaew

I just forwarded you and my three friend.

Should I open AC in this as it's not looking so ********

**Comment by devtestnet** [comment]
And just keep in your mind. Whatsoever is going around me is all done 100% by me but I was not know to this. Aage aage Dekho hota Hai Kya, wrong doing by knowing is @me d.Gusainn@gmail.com

Now you have to give complete explanation and one two one ☝️-  discussion. GoD knows what I went with me.

If I will be wrong then I am ready to accept it.

So before trade, swap and any illegal work got to be find done by anyone will BE INFRONT OF ME(deepak Gusain).

![image](https://user-images.githubusercontent.com/84843897/124292006-c9016a80-db72-11eb-8fd0-7dc44db683d0.png)![image](https://user-images.githubusercontent.com/84843897/124292007-c9016a80-db72-11eb-95a8-daf1723db8d8.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292010-c99a0100-db72-11eb-8854-b840a134e844.jpeg)
![image](https://user-images.githubusercontent.com/84843897/124292026-cdc61e80-db72-11eb-8eed-aa552c64f6b6.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292070-d4ed2c80-db72-11eb-85cb-fe71e7f18309.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292074-d585c300-db72-11eb-8d50-314b0c58df3b.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292092-d9194a00-db72-11eb-99c2-3ae3439a9f50.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292116-dd456780-db72-11eb-876b-80726f00c675.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292117-dd456780-db72-11eb-9831-0277e3f517ef.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292119-dd456780-db72-11eb-9c4c-93d9d3735e58.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292120-dd456780-db72-11eb-8c28-b74b05b1fd94.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292115-dd456780-db72-11eb-9c23-a1369c02664c.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292122-ddddfe00-db72-11eb-9400-f9d7a2b5c081.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292124-ddddfe00-db72-11eb-99f1-ea18b79c58d4.jpeg)![image](https://user-images.githubusercontent.com/84843897/124292125-ddddfe00-db72-11eb-99f9-f4ae3d731f6f.jpeg) followed by all

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
